cmd/agent/dist/conf.d/process_agent.yaml.default

```diff
@@ -0,0 +1,6 @@
+instances:
```

Comment:
I think this would start the process agent by default on all platform and not just OXS

Reply:
Why would that be the case? I thought only macOS uses this file to trigger a long run process.

Reply:
I also did a quick datadog-agent installation on a ubuntu host using our one-step installation script, I searched through the file system and didn't find the existence of `apm.yaml.default`

Reply:
the logic for the trace-agent currently is:

1. `apm.yaml.default` is copied by the invoke task: https://github.com/DataDog/datadog-agent/blob/fabdc7ef5f5fa66551892fc7b9f6f578f1230007/tasks/agent.py#L191 (you seem to be doing this already in this PR for the process agent)
2. the `datadog-agent` omnibus software def copies the file to `<agent_etc_path>/conf.d/`, along with all the other files in `./bin/agent/dist/conf.d/`: https://github.com/DataDog/datadog-agent/blob/fabdc7ef5f5fa66551892fc7b9f6f578f1230007/omnibus/config/software/datadog-agent.rb#L77
3. the `datadog-agent` omnibus software def removes the file `<agent_etc_path>/conf.d/apm.yaml.default` on the platforms where it shouldn't be shipped, i.e. all platforms but macOS: see for [linux](https://github.com/DataDog/datadog-agent/blob/fabdc7ef5f5fa66551892fc7b9f6f578f1230007/omnibus/config/software/datadog-agent-finalize.rb#L89) and [windows](https://github.com/DataDog/datadog-agent/blob/fabdc7ef5f5fa66551892fc7b9f6f578f1230007/omnibus/config/software/datadog-agent-finalize.rb#L41)

I believe the step you're currently missing in the PR for process-agent is `3.`. Feel free to do the exact same thing as what's done for the trace-agent.

Side note: ideally, we should consolidate the logic so that `1.` and `3.` are done in one step in the invoke task, but I don't think it's necessary to try and improve this now. We have a card in our backlog to improve this in the future.